docker-app/qfieldcloud/core/models.py

```diff
@@ -1283,6 +1303,55 @@ def shared_datasets_project(self) -> Project | None:
         except Project.DoesNotExist:
             return None
 
+    def last_package_job_for_user(self, user: User) -> PackageJob | None:
+        """Returns the last package job for the user.
+
+        Args:
+            user: The user to check for.
+
+        Returns:
+            The last package job for the user.
+        """
+        return (
+            self.jobs.for_user(user)  # type: ignore[attr-defined]
+            .filter(
+                type=Job.Type.PACKAGE,
+            )
+            .order_by("-created_at")
+            .first()
+        )
+
+    def last_package_jobs(self) -> JobQuerySet:
```

Comment:
```suggestion
    def lastest_package_jobs(self) -> JobQuerySet:
```
No strong opinion, I just think it is a bit more correct English?

Reply:
I guess so, thanks for spotting this.